package.json

```diff
@@ -23,7 +23,7 @@
     "test:lint-fix": "yarn test:stylelint && nps test.lint.fix",
     "test:lint": "yarn test:stylelint && nps test.lint",
     "test:unit": "rm -rf coverage && npm run test:jest",
-    "test": "yarn test:lint && yarn test:unit",
+    "test": "yarn test:lint && yarn test:unit && yarn test:jest",
```

Comment:
@DannyMcwaves this script now calls jest twice. I'm going to fix it and clean up other scripts, so no need to fix.